test/helpers/components/dynakube/dynakube.go

```diff
@@ -140,24 +141,20 @@ func (dynakubeBuilder Builder) Build() dynatracev1beta1.DynaKube {
 	return dynakubeBuilder.dynakube
 }
 
-func Apply(dynakube dynatracev1beta1.DynaKube) features.Func {
+func Create(dynakube dynatracev1beta1.DynaKube) features.Func {
```

Comment:
Just a quick input: wouldn't it be better to just return a dynakube here and apply it where the Create() is called?